### PR TITLE
TELCODOCS-271: 4.7 release note changes for MGMT recovery project

### PR DESCRIPTION
**Pull request by aireilly**
4.7 release notes for: https://issues.redhat.com/browse/TELCODOCS-271

### CODE REVIEW

**Comment by aireilly** [comment]
/LGTM was provided over email to @abhatt-rh by @n1r1 for the text of release note for MGMT-299 epic documenting **Fencing: Power-Based Health Check Remediation for Bare Metal Clusters**

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -678,6 +678,16 @@ In {product-title} 4.6, when the online flag in the BareMetalHost API is set to
...
 
+[id="ocp-4-7-machine-health-check-timer-example"]
+==== Updated machine health check timer example
+
+The machine health resource example described in xref:../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[About machine health checks] has been updated with a shorter health check timer value.
+
+[id="ocp-4-7-power-based-remediation-reprovisioning-unhealthy-node"]
+==== Reprovisioning unhealthy nodes if power-based remediation does not complete successfully
+
+Starting with {prod-title} 4.7, during power-based remediation if the power operations do not complete successfully, the bare-metal machine controller triggers the reprovisioning of the unhealthy node. The exception to this is if the node is a master node or a node that was provisioned externally.
```
Comment (bmcelvee) [inline release_notes/ocp-4-7-release-notes.adoc:689]: {prod-title} is not displaying in the preview. Did you mean {product-title}?

**Comment by bmcelvee** [comment]
One comment, otherwise LGTM! 

I did gave a cursory check through emails, and didn't see any change management notifications for this one. Since it's modifying GA'd release notes, it should go through the change management process.

**Comment by aireilly** [comment]
> One comment, otherwise LGTM!
> 
> I gave a cursory check through emails, and didn't see any change management notifications for this one. Since it's modifying GA'd release notes, it should go through the change management process.

Thanks Brandy! Actuallly haven't sent the email yet. Figured to get the Peer review done, and then send out the mail. Thanks :)

**Comment by aireilly** [comment]
Hi @bmcelvee, this is also ready to merge, and I havn't gotten any feedback from my change managment email. Can you merge this one too please?

**Pull request merged**